### PR TITLE
Apcon modules

### PR DESCRIPTION
**Pull request by davidlee-ap**
##### SUMMARY
Add apconos_command module

##### ISSUE TYPE

- Bugfix Pull Request
- Docs Pull Request
- Feature Pull Request
- New Module Pull Request

##### ADDITIONAL INFORMATION

```paste below

```

### CODE REVIEW

File: lib/ansible/module_utils/network/apconos/apconos.py [hunk 1]
```diff
@@ -0,0 +1,114 @@
...
+        out = connection.get(**cmd)
+        responses.append(to_text(out, errors='surrogate_then_replace'))
+
+    return responses
+
+
+def load_config(module, config):
+    try:
+        conn = get_connection(module)
+        conn.get('enable')
```
Comment (ganeshrn) [inline lib/ansible/module_utils/network/apconos/apconos.py]: Shouldn't this command be added when there is privilege escalation enable with ansible.
Refer: https://github.com/ansible/ansible/blob/devel/lib/ansible/plugins/terminal/ios.py#L71
https://docs.ansible.com/ansible/latest/network/user_guide/platform_ios.html#using-cli-in-ansible

File: lib/ansible/plugins/cliconf/apconos.py [hunk 2]
```diff
@@ -0,0 +1,75 @@
...
+    def parse_version(self, data):
+        return ""
+
+    def parse_model(self, data):
+        return ""
+
+    @enable_mode
+    def get_config(self, source='running', format='text'):
+        cmd = b''
+        return self.send_command(cmd)
```
Comment (ganeshrn) [inline lib/ansible/plugins/cliconf/apconos.py]: does sending a empty command return the configuration from the device?

File: lib/ansible/plugins/cliconf/apconos.py [hunk 3]
```diff
@@ -0,0 +1,75 @@
...
+    def edit_config(self, command):
+        for cmd in chain([b'configure terminal'], to_list(command), [b'end']):
+            self.send_command(cmd)
+
+    def get(self, command, prompt=None, answer=None, sendonly=False, check_all=False):
+        return self.send_command(command=command, prompt=prompt, answer=answer, sendonly=sendonly, check_all=check_all)
+
+    def get_capabilities(self):
+        result = {}
+        return json.dumps(result)
```
Comment (ganeshrn) [inline lib/ansible/plugins/cliconf/apconos.py]: `result` is empty and no capability is returned. Is this expected? You might want to call `get_device_info` here to get the device capabilities.

File: lib/ansible/plugins/doc_fragments/apconos.py [hunk 4]
```diff
@@ -0,0 +1,16 @@
+# -*- coding: utf-8 -*-
+
+# Copyright: (c) 2019, David Li<@davidlee-ap>
```
Comment (ganeshrn) [inline lib/ansible/plugins/doc_fragments/apconos.py]: This file is not required

File: lib/ansible/plugins/cliconf/apconos.py (hunk 2 above)
Comment (davidlee-ap) [reply lib/ansible/plugins/cliconf/apconos.py]: I ignored this function. No configuration can be returned for now.

File: lib/ansible/plugins/cliconf/apconos.py (hunk 3 above)
Comment (davidlee-ap) [reply lib/ansible/plugins/cliconf/apconos.py]: Yes, I replaced it with ge_device_info.

File: lib/ansible/module_utils/network/apconos/apconos.py (hunk 1 above)
Comment (davidlee-ap) [reply lib/ansible/module_utils/network/apconos/apconos.py]: I ignored privilege escalation  for now.

File: lib/ansible/plugins/doc_fragments/apconos.py (hunk 4 above)
Comment (davidlee-ap) [reply lib/ansible/plugins/doc_fragments/apconos.py]: The file has been removed.

**Review by ganeshrn — APPROVED** [review]

**Pull request merged**